src/Integration.Vsix.UnitTests/SonarLintTagger/AccumulatingIssueConsumerTests.cs

```diff
@@ -184,6 +201,15 @@ private static ITextSnapshot CreateSnapshot(int lineCount)
                 }
             };
 
+        private static IAnalysisIssue CreateFileLevelIssue()
+        {
+          return new DummyAnalysisIssue
+            {
+                PrimaryLocation = new DummyAnalysisIssueLocation()
```

Comment:
Minor: this would be clearer and more robust if it explicitly set the `TextRange` to `null` (it would be a strong hint that `TextRange` being `null` is what makes it a file-level issue, and the test won't break if we later added a default constructor to `DummyAnalysisIssueLocation` that changed its behaviour).